lib/facilities/gis_configuration.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'common/client/middleware/response/raise_error'
+require 'common/client/middleware/response/facility_validator'
+require 'common/client/middleware/response/facility_parser'
+
+module Facilities
+  class GisConfiguration < Common::Client::Configuration::REST
+    def base_path
+      Settings.locators.gis_base_path
+    end
+
+    def service_name
+      'FL'
```

Comment:
Seems a little strain this isn't a constant, but not a huge issue.

Reply:
True, but this follows the standard that is used in all of our configuration classes.